app/presenters/place_presenter.rb

```diff
@@ -11,4 +11,19 @@ class PlacePresenter < ContentItemPresenter
       details[key.to_s] if details
     end
   end
+
+  def initialize(content_item, places = nil)
+    @content_item = content_item
+    @places = places if places
```

Comment:
there's no need for the `if` guard here - it makes very little difference to the program logic if an instance variable is initialised to `nil`